framework/src/Volo.Abp.BlazoriseUI/Components/AbpExtensibleDataGrid.razor

```diff
@@ -112,7 +112,7 @@
                                                 Sortable="@column.Sortable"
                                                 Displayable="column.Visible">
                                     <DisplayTemplate>
-                                        @(GetConvertedFieldValue(context, column))
+                                        @((MarkupString)GetConvertedFieldValue(context, column))
```

Comment:
```
    new TableColumn
    {
        Title = L["PrivateMessageCreationTime"],
        Sortable = true,
        Data = nameof(PrivateMessageViewModel.CreationTime),
        ValueConverter = data=>FormatCell(data,"CreationTime")
    },
...
    private static string FormatCell(object data,string name)
    {
        var row = data.As<PrivateMessageViewModel>();
        var cell = row.ReadTime;
        var valueStr = name switch
        {
            "Title" => row.Title,
            "CreationTime" => row.CreationTime.ToString("F"),
            _ => row.FromUserName
        };

        return cell == null ? "<span class='bold'>" + valueStr + "</span>" : valueStr;
    }
```
https://github.com/SyrnaAbp/Syrna.PrivateMessaging/blob/3a2fc8074135b72a9256070b8956d00172dd2512/modules/src/Syrna.PrivateMessaging.Blazor/Pages/PrivateMessaging/PrivateMessages/Inbox.razor.cs#L151

Using Html tags for cell formatting in ValueConverter. We can use the method for Blazor datagrid row/cell formatting or displaying value with html. 

If dont use MarkupString, it will be show non rendered html.

Reply:
Thank you for your explanation.